package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
I would prefer it if you didn't commit package-lock changes in an unrelated PR. If you'll have any trouble removing the file from the commit, let me know and I'll do it before merging

Reply:
I tried to reset but there is still a small diff is being shown

Reply:
I've replied below wrt to the diff